sparkly/reader.py

```diff
@@ -214,6 +219,71 @@ def mysql(self, host, database, table, port=None, parallelism=None, options=None
 
         return self._basic_read(reader_options, options, parallelism)
 
+    def kafka(self,
+              brokers,
+              topics=None,
+              offset_ranges=None,
+              key_deserializer=None,
+              value_deserializer=None,
+              schema=None,
+              parallelism=None,
+              options=None):
+        """Creates dataframe from specified set of messages from Kafka topic.
+
+        If `offset_ranges` parameter is omitted it will auto-discover.
+        If `schema` parameter is specified the Dataframe with this schema will be returned,
+        else the RDD will be returned.
+        The `schema` parameter, if specified, should contain two top level fields:
+        `key` and `value`.
+
+        Args:
+            brokers (list): List of kafka brokers.
+            topics (list[str]): List of kafka topics to read from.
+            offset_ranges (list[(str, int, int, int)]): List of partition ranges
```

Comment:
- how do we match multiple topics to `offset_ranges`?
- do we really need multiple topics here? User can always do:
```
df_a = hc.read_ext.kafka(topic='a')
df_b = hc.read_ext.kafka(topic='b')
df_a.union(df_b)
```